tests/foreman/api/test_product.py

```diff
@@ -399,3 +403,46 @@ def test_positive_filter_product_list(self):
         self.assertGreater(len(rh_products), 1)
         self.assertNotIn(product.name, [prod.name for prod in rh_products])
         self.assertIn('Red Hat Beta', [prod.name for prod in rh_products])
+
+    @tier2
+    def test_positive_assign_http_proxy_to_products(self):
+        """Assign http_proxy to Products.
+
+        :id: c9d23aa1-3325-4abd-a1a6-d5e75c12b08a
+
+        :expectedresults: HTTP Proxy is assigned to all repos present
+            in Products.
```

Comment:
Can you add a product with more than one repo to test?

The claim is that proxy is assigned to "all" repos, but with current approach it might as well be assigned to first repo, last repo or random repo - result is the same, as product has only one.

Reply:
@mirekdlugosz Updated the PR, I have now added two repos in each product.